src/utils/self_test/self_test.c

```diff
@@ -345,12 +562,48 @@ static void print_results(struct st_latency *latencies,
 	else
 		printf("\tRPC Bandwidth (MB/sec): %.2f\n",
 		       bandwidth / (1024.0F * 1024.0F));
-	printf("\tRPC Throughput (RPCs/sec): %.0f\n", throughput);
 
+	printf("\tRPC Throughput (RPCs/sec): %.0f\n", throughput);
 
+	snprintf(new_key_name, sizeof(new_key_name), "%s-@:*-bw", master);
+	ConfigAddInt(Ocfg, section_name, new_key_name,
+		     bandwidth / (1024.0F * 1024.0F));
+	snprintf(new_key_name, sizeof(new_key_name), "%s-@:*-tp", master);
+	ConfigAddInt(Ocfg, section_name, new_key_name, throughput);
+
+#undef PRINT_ALL
```

Comment:
why is undef here? this #ifdef PRINT_ALL should probably be option to self-test to enable more verbose output instead of compile time

Reply:
I have modified the code, when printing header information to use D_DEBUG.  However, I kept it as a compile time for printing out individual values.  During debugging stage, the numbers of runs are small (i.e. 10-20).  However, the standard size for evaluation is up to 1000.  The size would swamp out the debug output to make it un-usable.  I would rather remove this printf.